pkgs/misc/emulators/mednafen/default.nix

```diff
@@ -26,6 +25,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
+    version = "0.9.38.7";
```

Comment:
No motivation specially, I use it because it is on documentation :)
There is a big discussion on this issue, but I particularly prefer that way - It's easier to parse IMHO.